pages/jobs/apiture-api-architect.md

```diff
@@ -0,0 +1,81 @@
+---
+title: 'API Architect'
+date: 22/06/2021
+category: job category
+closingOn: 22/08/2021
```

Comment:
date must be `American` style (at least I always considered it used only in US). So basically month first, day later
```suggestion
closingOn: 08/22/2021
```

now you are getting an invalid date https://deploy-preview-285--asyncapi-website.netlify.app/jobs

Reply:
Thanks. I simply followed the template from the Post It Now link on https://www.asyncapi.com/jobs which used DD/YY/MMMM .
I can update that link as well.